CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+## 1.0.0
+
+### Features
+
+Add self-hosted API Server support.
```

Comment:
Isn't it better to use "Added"

Reply:
Thanks! It seems we current have two styles for change log. It would be nice if we could come to an agreement. For now, we use present tense in change log, aligning with the rest of open source projects.

Reply:
yes. following the link you mentioned in the chat, I think the current style is good.
https://github.com/facebook/react/blob/main/CHANGELOG.md